pnpm-lock.yaml

```diff
@@ -404,6 +404,9 @@ importers:
       '@angular/ssr':
         specifier: 19.0.5
         version: 19.0.5(whst2uom2sykn2ay52lywk7vqm)
+      '@ng-rspack/build':
```

Comment:
This should get removed. IMO the repository should not install the package, but only use it for local verdaccio install or directly from the dist folder. (happy to hear other opinions)

Was it a left over?

Reply:
It's an intended change since we use ts project references and package manager's workspaces to link packages. This is already the case for packages, and this PR introduces the same behavior for apps.

![image](https://github.com/user-attachments/assets/ce869c72-4e01-4077-982f-bfa6d78c2560)

Please take a look at [the docs about ts project linking](https://nx.dev/concepts/typescript-project-linking#create-a-new-nx-workspace-using-project-references).

Reply:
Got it! Thx for clarifying.